src/GitHub.VisualStudio/Menus/BlameLink.cs

```diff
@@ -0,0 +1,42 @@
+﻿using GitHub.Services;
+using GitHub.VisualStudio.UI;
+using NullGuard;
+using System;
+using System.Windows;
+
+namespace GitHub.VisualStudio.Menus
+{
+    public class BlameLink : LinkMenuBase, IDynamicMenuHandler
+    {
+
+        public BlameLink(IGitHubServiceProvider serviceProvider)
+            : base(serviceProvider)
+        {
+        }
+
+        public Guid Guid => GuidList.guidContextMenuSet;
+        public int CmdId => PkgCmdIDList.blameCommand;
+
+        public async void Activate([AllowNull]object data = null)
+        {
+            var isgithub = await IsGitHubRepo();
+            if (!isgithub)
+                return;
+
+            var link = await GenerateLink(true);
```

Comment:
Minor nit, but in terms of API design, I generally avoid booleans in this situation. Looking at this, it's not immediately clear what `true` means. Nor what `false`  means.

One way to improve it is to use a named parameter `GenerateLink(blame: true)`. Even so, it's not clear what false would mean in cases where `GenerateLink` is called.

Typically in this case, I think rather than using a boolean, an Enum might make it clear. Especially if there will be other kinds of link. Then it could say `GenerateLink(LinkType.Blame)` or `GenerateLink(LinkType.Blob)`.

If you don't anticipate that, then I'm fine with just specifying the named parameter.